.buildkite/scripts/buildkite-integration-tests.sh

```diff
@@ -26,15 +26,15 @@ fi
 asdf install
 
 echo "~~~ Running integration tests as $USER"
-echo "~~~ Integration tests: ${GROUP_NAME}"
 
 go install gotest.tools/gotestsum
 gotestsum --version
 
-PACKAGE_VERSION="$(cat .package-version)"
-if [[ -n "$PACKAGE_VERSION" ]]; then
-    PACKAGE_VERSION=${PACKAGE_VERSION}"-SNAPSHOT"
-fi
+# Parsing version.go. Will be simplified here: https://github.com/elastic/ingest-dev/issues/4925
+AGENT_VERSION=$(grep "const defaultBeatVersion =" version/version.go | cut -d\" -f2)
+AGENT_VERSION="${AGENT_VERSION}-SNAPSHOT"
+export AGENT_VERSION
+echo "~~~ Agent version: ${AGENT_VERSION}"
```

Comment:
I see this in two places, do you mind moving this logic to a make goal or something else? Otherwise, it's hard to maintain.

For instance, the APM Server uses:

- https://github.com/elastic/apm-server/blob/42e0cbf5b049a92b5536f34090b15f5eb0a7167e/go.mk#L11
- and https://github.com/elastic/apm-server/blob/42e0cbf5b049a92b5536f34090b15f5eb0a7167e/Makefile#L146-L149

Therefore the call is as simple as this https://github.com/elastic/apm-server/blob/42e0cbf5b049a92b5536f34090b15f5eb0a7167e/.buildkite/scripts/dra.sh#L39

Reply:
What is the second place? If it's the PS file for Windows then unfortunately it's hard to avoid. 
I also added a comment above. The mentioned [issue](https://github.com/elastic/ingest-dev/issues/4925) will replace it with `AGENT_VERSION=$(cat .agent-version)`. 
I would like to ask you a QQ about your example if you don't mind. Wouldn't it be easier to create a similar `.version` file and embed it to the `version.go`? Then getting the version will be as simple as it mentioned above

Reply:
> What is the second place? If it's the PS file for Windows then unfortunately it's hard to avoid(Possible but the solution is controversial).

Yep

> Wouldn't it be easier to create a similar .version file and embed it to the version.go? 

I think so, thanks for bringing this to my attention, if there is a plan, then all good.